### DISCUSSION TITLE
How to get a new badge named "YOLO"?

### DISCUSSION THREAD

**Discussion by lhh1171** [community/community#18846] [Other Feature Feedback, Questions, & Ideas] [answered]
#How to get a new badge named "yoLo"?
Can you elaborate it

**Comment by melvincwng** [comment, marked as answer]
@lhh1171, to get YOLO achievement badge, merge your PR into a public repository without doing a code review.

Reference: https://github.com/Schweinepriester/github-profile-achievements

Hope this clarifies

**Reply by lhh1171** [reply]
got it

**Comment by MartynSorin** [comment]
It's clear
Thank you
